core/transaction.go

```diff
@@ -1,9 +1,66 @@
 package core
 
 import (
+	"math/big"
+
 	"github.com/NethermindEth/juno/core/felt"
 )
 
+type Event struct {
+	Data []*felt.Felt
+	From *felt.Felt
+	Keys []*felt.Felt
+}
+
+type L1ToL2Message struct {
+	From     string
```

Comment:
Is the `From` field an Ethereum address? If so, I assume the `To` field in `L2ToL1Message` is also an Ethereum address. Is that correct?

Reply:
Yes, that is correct

Reply:
Great! In this case, we could use a more precise type than `string`. Maybe `common.Address` from geth or even `[20]byte`?

Reply:
Alright. I think `common.Address` is better. That's what is used to represent Ethereum addresses.